src/jdk.internal.vm.ci/share/classes/jdk/vm/ci/hotspot/HotSpotConstantPool.java

```diff
@@ -396,18 +396,6 @@ private int getNameAndTypeRefIndexAt(int rawIndex, int opcode) {
         return compilerToVM().lookupNameAndTypeRefIndexInPool(this, rawIndex, opcode);
     }
 
-    /**
-     * Gets the name of a {@code JVM_CONSTANT_NameAndType} constant pool entry referenced by another
-     * entry denoted by {@code which}.
-     *
-     * @param rawIndex rewritten index in the bytecode stream
-     * @param opcode the opcode of the instruction for which the lookup is being performed
-     * @return name as {@link String}
-     */
-    private String getNameOf(int rawIndex, int opcode) {
```

Comment:
This method, and `getSignatureOf()` seem to be unused but is it possible that they could have use in the future?

Reply:
These were private methods that had a single caller. All they did was to forward the call to CompilerToVM, so I removed them so that I didn't need to worry about renaming their parameters.

In the future, after we get rid of `which` and `cpci`, maybe we can refactor the code to have better organization, and possibly bring back these methods.